src/test/java/pet/FindPetNegativeTest.java

```diff
@@ -0,0 +1,42 @@
+package pet;
+
+import client.PetClient;
+import data.PetInfo;
+import dto.requests.pet.ResponseInfo;
+import io.restassured.response.Response;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+
+public class FindPetNegativeTest {
+
+    private static final int DEFUNCT_PET_ID = 666666;
```

Comment:
What if pet with such id will be created by another user? We have to come up with another way to verify this case